dev-docs/modules/consentManagement.md

```diff
@@ -0,0 +1,179 @@
+3. With a valid consent string, we will incorporate this data into the auction objects (for adapters to collect) and then allow the auction to proceed.
+
+There are timeout settings in place in the module to permit this interaction with the CMP a specified length of time to operate before it's unacceptable or assumed an issue has occurred.  
+
+When either this timeout occurs or if an error from the CMP is thrown, one of two options are taken; either:
+
+1. The auction is canceled outright.
+2. The auction proceeds without the user's consent information.  
+
+Though these options are mutually exclusive, they are configurable by the publisher via the site's implementation of the prebid code (see further below for details) so that they can be used in the proper scenarios for that site/audience.
+
+
+## Page integration
+
+To utilize this module, a separate CMP needs to be implemented onto the site to interact with the user and obtain their consent choices.  
+
+The actual implementation details of this CMP are not covered by this page; any questions on that implemenation should be referred to the CMP in question.  However, we would recommend to have the CMP's code located before the prebid code in the head of the page, in order to ensure their framework is implemented before the prebid code starts to execute.
+
+The module currently supports the following CMPs:
+
+* AppNexus (appnexus)
+
+Once the CMP is implemented, simply include the module in your build and add a consentManagement object in the setConfig() call.  Adapters that support this feature will be able to retrieve the consent information and incorporate it in their requests.
+
+{: .table .table-bordered .table-striped }
+| Param | Type | Description | Example |
+| --- | --- | --- | --- |
+| cmp | `string` | The ID for the CMP in use on the page.  Default is `'appnexus'` | `'appnexus'` |
+| consentRequired | `boolean` | An override type setting to indicate if GDPR consent is required or not.  See note in regards to default. | `true` or `false` |
+| timeout | `integer` | Length of time (in milliseconds) to allow the CMP to perform its tasks before aborting the process. Default is `10000` | `10000` |
+| allowAuctionWithoutConsent | `boolean` | A setting to determine what will happen when obtaining consent information from the CMP fails; either allow the auction to proceed (**true**) or cancel the auction (**false**). Default is `true` | `true` or `false` |
+
+* Note - There are some technologies to determine if a given request is in scope of GDPR or not.  While this technology is not part of the consentManagement module (nor prebid), some adapters may have this technology available.  If they do, they have the opportunity to set their own default value for the `consentRequired` field.  If you are using a GDPR supported adapter that has this capability, simply do **not** include this field in your config to let the corresponding adapter(s) set their value.
```

Comment:
It's not the adapter that has the ability to determine GDPR scope... probably the SSP/exchange/bidder. Would suggest wording more like this:


Note - Some SSPs can determine whether a given request is in GDPR scope or not. If the page specifies `consentRequired`, it will override any dynamic determination and force the bidders to use this override value -- i.e. tell the SSP whether the consent string must be enforced for this user. Each bidder adapter supporting GDPR will default to the proper setting for the backend SSP, so it's not recommended to set this value unless override is what's desired.

Reply:
Thanks for the feedback, I'll make the changes as recommended.